factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/homepage.md

```diff
@@ -0,0 +1,21 @@
++++
+draft = false
+title = "Home page"
+description = ""
+date = "2017-04-28T18:36:24+02:00"
+creatordisplayname = "Valere JEANTET"
+creatoremail = "valere.jeantet@gmail.com"
+lastmodifierdisplayname = "Valere JEANTET"
+lastmodifieremail = "valere.jeantet@gmail.com"
+tags = ["tag1","tag2"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)